api/project/models/project.js

```diff
@@ -5,4 +5,112 @@
  * to customize this model
  */
 
-module.exports = {};
+const axios = require('axios');
+const _ = require('lodash');
+
+module.exports = {
+  lifecycles: {
+    async beforeUpdate(params, data) {
+      try {
+        let projectBeforeModification = await strapi
+          .query('project')
+          .findOne(params, ['heroImage', 'heroImage.formats']);
+        let concatinated = [
+          ...projectBeforeModification.openPositions,
+          ...data.openPositions,
+        ]
+          .filter(
+            (v, i, a) =>
+              a.findIndex(
+                (t) =>
+                  t.id === v.id &&
+                  t.title === v.title &&
+                  t.description === v.description &&
+                  t.isHidden === v.isHidden
+              ) === i
+          )
+          .slice(projectBeforeModification.openPositions.length);
+        let deletedFields = _.differenceBy(
+          projectBeforeModification.openPositions,
+          data.openPositions,
+          'id'
+        );
+
+       
+        if (
+          !_.isEqual(
+            projectBeforeModification.openPositions,
+            data.openPositions
+          ) && process.env.NODE_ENV == 'production'
+        ) {
+          await axios.post(
+            process.env.OPEN_POSITIONS_DISCORD_WEBHOOK,
+            {
+              username: `${projectBeforeModification.title} Notifier`,
+              avatar_url:
+                'https://avatars.githubusercontent.com/u/53379976?s=200&v=4',
+              embeds: [
+                {
+                  title: `${projectBeforeModification.title} Project Page`,
+                  url: `https://devlaunchers.org/projects/${projectBeforeModification.slug}`,
+                  description: `*someone* **__UPDATED__** the open roles => [Link to the strapi page](${process.env.FRONTEND_URL}/admin/plugins/content-manager/collectionType/application::project.project/${params.id})`,
+                  color: 15258703,
+                  fields: [
```

Comment:
No, this is only going to show up in the admins private channel in discord and they're going update volunteer match list according to the changes happened in strapi.
To answer your second question I'd say not really because the admin team said they're only wanting those specific changes occurring on the data.